tests/test_base_device.py

```diff
@@ -239,11 +221,12 @@ def test_backend_wire_validation(self, backend):
         with pytest.raises(ValueError, match="supports maximum"):
             QiskitDevice2(wires=500, backend=backend)
 
+    @pytest.mark.skip(reason="Options handling not decided on yet")
     def test_setting_simulator_noise_model(self):
         """Test that the simulator noise model saved on a passed Options
         object is used to set the backend noise model"""
 
-        options = Options()
+        options = {}
```

Comment:
Has the `Options` changed as well?

Reply:
`Options` has been deleted due to changes by Qiskit. [See this](https://docs.quantum.ibm.com/api/migration-guides/v2-primitives#steps-to-migrate-to-estimator-v2).

Now they want you to specify options with either some kind of dictionary using options={"k": v} where they'll parse that object to get keyword arguments or just use keyword arguments to specify options.